lib/xpm_ruby/connection.rb

```diff
@@ -16,9 +16,16 @@ def get(endpoint:, params: nil)
 
       case response.status
       when 401
-        hash = Ox.load(response.body, mode: :hash_no_attrs, symbolize_keys: false)
+        detail = JSON.parse(response.body)["Detail"]
```

Comment:
Ha so a 401 returns JSON? Crazy!

Reply:
Haha yeah it was previously returning `HTML` for `workflowmax` urls and now `JSON` for `practicemanager` ones.